.gitignore

```diff
@@ -12,3 +12,9 @@ ssl/serial*
 
 # downloaded by playbook
 library/lvg.py
+
+#mac
+.DS_Store
+
+#intellij
+.idea
```

Comment:
we don't want to include any OS- or editor-specific ignores in our .gitignore, this should be only for the project. These should go in your [global .gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)